activity/covid19/covid19_schema

```diff
@@ -1,30 +1,30 @@
 {
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
```

Comment:
shouldn't this point to refactor branch?

Reply:
yes. thanks, will change.